README.md

```diff
@@ -564,7 +564,6 @@ Clients for commercial social platforms that had their API access cut off in a w
 - [Szyszka](https://github.com/qarmin/szyszka) - Fast bulk file renamer `#rust` `#gtk4`.
 - [Ticket Booth](https://github.com/aleiepure/ticketbooth) - Application to keep track of TV series/movies with TMDB's API `#python` `#gtk4` `#libadwaita`.
 - [TV Series Renamer](https://github.com/mmstick/tv-renamer) - TV series renaming application that support adding titles to episodes `#rust` `#gtk3`.
-- [Workspaces](https://github.com/DevAlien/workspaces) - Desktop workpaces for elementaryOS `#vala` `#gtk3` `#granite`.
```

Comment:
Why should this application be removed?

Reply:
Even if the code hasn't been updated for a while, there is no alternative for this feature on the list.

Reply:
Because it no longer makes sense, the multitasking view feature in Pantheon closes all user needs from this application and is implemented at the shell level